packages/react-dom-bindings/src/events/plugins/FormActionEventPlugin.js

```diff
@@ -92,6 +96,19 @@ function extractEvents(
   }
   const formInst = maybeTargetInst;
   const form: HTMLFormElement = (nativeEventTarget: any);
+
+  if (!disableInputAttributeSyncing) {
+    for (let i = 0; i < form.elements.length; i++) {
+      const input: HTMLInputElement = (form.elements.item(i): any);
+      if (
+        input.nodeName.toLowerCase() === 'input' &&
```

Comment:
```suggestion
        input.nodeName === 'INPUT' &&
```

Reply:
I just saw this particular format in another place and decided to repeat it (_but there's no point in this_)

https://github.com/facebook/react/blob/main/packages/react-dom-bindings/src/client/inputValueTracking.js